src/Symfony/Bundle/FrameworkBundle/Command/CachePoolClearCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+
+/**
+ * Clear cache pools.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class CachePoolClearCommand extends ContainerAwareCommand
```

Comment:
> you could add this question to most PRs

That's true, but Github provides a saved reply feature, it's not a problem for me.

> An in fact here, no other command is final

I feel it doesn't make my question less reasonable.
If you don't add `final`, you agree with BC for those who wants to extend this command?
I don't think so, because, you know, nobody in their right mind would reuse command in this way. Thus, you won't keep this kind of BC for commands, why you don't want to make it evident?

After all, does it make sense to make difference between classes? "Oh, these classes are `final` in fact, but we don't mark them by tradition".